indico/htdocs/sass/modules/abstracts/_email_templates.scss

```diff
@@ -55,16 +55,59 @@
     }
 }
 
-.email-template-list {
-    list-style-type: none;
-    padding-left: 0;
-    width: 800px;
+#email-template-manager {
+    max-width: 800px;
+
+    .email-template-container {
+        padding: 0 0 1em 1em;
+
+        .email-template-notifications {
+            width: 800px;
+            max-height: 800px;
+            overflow: auto;
+
+            .email-template-list {
+                list-style-type: none;
+                padding-left: 0;
+                padding-right: 20px;
+
+                .email-template {
+                    margin-bottom: 10px;
+
+                    .email-preview {
+                        display: none;
+
+                        .email-subject {
+                            padding-bottom: 10px;
+                        }
 
```

Comment:
do we really need that much nesting? you don't need to replicate the DOM's nesting in CSS unless things would be ambiguous otherwise